client/network/src/lib.rs

```diff
@@ -272,10 +272,7 @@ pub use protocol::{
 	sync::{StateDownloadProgress, SyncState, WarpSyncPhase, WarpSyncProgress},
 	PeerInfo,
 };
-pub use service::{
-	IfDisconnected, NetworkService, NetworkWorker, NotificationSender, NotificationSenderReady,
-	OutboundFailure, RequestFailure, Signature,
-};
+pub use service::*;
```

Comment:
I much prefer when the list of items that are publicly-exported is explicit.

Reply:
Same

Reply:
An I prefer using `pub(crate)` and `pub(super)` for symbols I do not want to export from the crate. Also, I do not drink beer. All preferences. BUT, because you are senior here, I will play by the rules of course and keep the `use` noise level up. :stuck_out_tongue: